packages/salesforcedx-vscode-lwc/test/vscode-integration/commands/forceLightningLwcPreview.test.ts

```diff
@@ -169,21 +166,8 @@ describe('forceLightningLwcPreview', () => {
   const notLwcModulePathUri = URI.file(notLwcModulePath);
   const nonExistentPath = path.join(root, 'foo');
   const nonExistentPathUri = URI.file(nonExistentPath);
-  let showQuickPickStub: sinon.SinonStub<
-    [
-      vscode.QuickPickItem[] | Thenable<vscode.QuickPickItem[]>,
-      (vscode.QuickPickOptions | undefined)?,
-      (vscode.CancellationToken | undefined)?
-    ],
-    Thenable<vscode.QuickPickItem | undefined>
-  >;
-  let showInputBoxStub: sinon.SinonStub<
-    [
-      (vscode.InputBoxOptions | undefined)?,
-      (vscode.CancellationToken | undefined)?
-    ],
-    Thenable<string | undefined>
-  >;
```

Comment:
These were causing issues.  We usually don't include types in our stubs so I just removed it.